.travis.yml

```diff
@@ -37,6 +37,8 @@ env:
    - DESTINATION="OS=10.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="NO"
    - DESTINATION="OS=10.1,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="YES" POD_LINT="NO"
 
+   - DESTINATION="OS=10.2,name=iPhone 7" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
```

Comment:
Ah, it's unable to find it due to a duplicate simulator for the 10.2 image. I'll use their suggested workaround until Travis fixes it on their end. 

https://github.com/travis-ci/travis-ci/issues/7031